test/pool_event_handler.test.ts

```diff
@@ -55,12 +61,20 @@ jest.mock('../src/utils/config.js', () => {
     APP_CONFIG: config,
   };
 });
+jest.mock('child_process');
 
 describe('poolEventHandler', () => {
   let db: AuctioneerDatabase;
   let mockedSorobanHelper = new SorobanHelper() as jest.Mocked<SorobanHelper>;
   let mockedUpdateUser = updateUser as jest.MockedFunction<typeof updateUser>;
-  let poolEventHandler: PoolEventHandler;
+  let mockedSendEvent = sendEvent as jest.MockedFunction<typeof sendEvent>;
+  const mockedWorkerProcess = {
+    send: jest.fn(),
+    on: jest.fn(),
+    // Add any other methods or properties you need to mock
+  } as unknown as ChildProcess;
```

Comment:
can you just write

```
let mockedWorker = new ChildProcess() as jest.Mocked<ChildProcess>;
```

Reply:
actually, reading the test closer this doesn't even need to be mocked. You mock `sendEvent` in `messages.ts`.

you can omit the jest mock on child process, and just write:

```
let fakeWorker = { name: "worker" } as ChildProcess;
```

the inclusion of a prop is just to ensure the correct object is being passed to `sendEvent`.